doc/pr2_tutorials/planning/src/doc/move_group_interface_tutorial.rst

```diff
@@ -23,7 +23,6 @@ Temporary PR2 on Kinetic Instructions
 
 You will also need a **pr2_moveit_config** package to run this tutorial. Currently this is unreleased in ROS Kinetic but the following is a temporary workaround::
 
-  git clone https://github.com/PR2/pr2_common.git -b kinetic-devel
```

Comment:
I don't think this is a good idea because the last release for pr2_common [was in 2016](https://github.com/PR2/pr2_common/releases) and there are many fixes since then. I [requested a few months ago](https://github.com/PR2/pr2_common/issues/265) a new release to no avail.